### PR TITLE
Style cleanups to meteor

### PR DESCRIPTION
**Pull request by bradcray**
I took a style pass over meteor-parallel as follows:

* wordsmithed comments slightly
* reformatted comments that wrapped 80 columns to emacs fill-column width
* promoted vars to consts and consts to params when possible/reasonable
* moved declarations to point of first definition when possible
* promoted '5' and '10' to params 'boardWidth' and 'boardHeight' and
  used where they seemed obviously OK (may be other uses as well)
* added spaces between arrays of arrays '[1..3] [1..5]' vs. '[1..3][1..5]'
* chained multiple consecutive var/const/param declarations to share keywords
* added some vertical whitespace
* reformatted 'if(expr)' as 'if expr'
* removed 'module meteor' and moved everything to file scope
* replaced some curly brackets with keywords when it seemed appropriate/reasonable
* pushed all global variables to the top of the file or into procedure scopes

### CODE REVIEW

**Comment by bradcray** [comment]
@lydia-duncan : Would you review to see if you have any objections or concerns?  I'm afraid that this isn't going to be easy because of my removing 'module meteor' -- so it might make sense to review individual commits.

File: test/studies/shootout/meteor/kbrady/meteor-parallel.chpl [hunk 1]
```diff
@@ -300,9 +300,9 @@ proc fillContiguousSpace(board, indx) {
 proc firstEmptyCell(cell, in min) {
   while (min == cell[0] || min == cell[1] ||
          min == cell[2] || min == cell[3] ||
-         min == cell[4]) {
+         min == cell[4]) do
```
Comment (lydia-duncan) [inline test/studies/shootout/meteor/kbrady/meteor-parallel.chpl]: Personally I prefer having brackets when the block enclosed spans multiple lines, even if it only encloses one outer statement.  I think it makes things clearer.  I wouldn't make you revert this commit unless others in the group agreed, though.

File: test/studies/shootout/meteor/kbrady/meteor-parallel.chpl [hunk 2]
```diff
@@ -344,11 +348,11 @@ module meteor {
     for row1 in 0..31 {
       for row2 in 0..31 {
         for row3 in 0..31 {
-          var idx = row1+(row2*32)+(row3*1024);
+          const idx = row1+(row2*32)+(row3*1024);
```
Comment (lydia-duncan) [inline test/studies/shootout/meteor/kbrady/meteor-parallel.chpl]: Not sure how I missed this, but I'd be tempted to add spaces between the "+" and each operand.

**Comment by lydia-duncan** [comment]
Seems good to me!

**Comment by bradcray** [comment]
Hi Lydia --

I was on the fence for some of those single-statement-is-a-multiple-statement cases and wouldn't object to reverting them.  Some of the others were more obvious "this seems like overkill" cases, though.  I'll try to revert some of the former cases and add the spaces around the pluses today, but if I time out, I may merge this and ask if you could do them before submitting this (either tomorrow or early next week) -- would that be OK?

**Comment by lydia-duncan** [comment]
That sounds like a good plan!

File: test/studies/shootout/meteor/kbrady/meteor-parallel.chpl (hunk 1 above)
Comment (bradcray) [inline test/studies/shootout/meteor/kbrady/meteor-parallel.chpl]: Oh but wait, for this specific case the block doesn't span multiple lines... I thought you were referring to cases like:

    forall ... do
      for ... {
         ... lots of lines ...
     }

which were the ones I was on the fence about.  Can you clarify?

**Comment by bradcray** [comment]
Also, Lydia -- Do you think we're ready to promote this to the release (I do), and is it OK if I do that in this PR as well?

File: test/studies/shootout/meteor/kbrady/meteor-parallel.chpl (hunk 1 above)
Comment (lydia-duncan) [inline test/studies/shootout/meteor/kbrady/meteor-parallel.chpl]: No, you had it right the first time, I was just confusing about where I put the note (as I foolishly assumed Ben and I'd caught all the single line ones)

Comment (lydia-duncan) [inline test/studies/shootout/meteor/kbrady/meteor-parallel.chpl]: Sorry to be confusing!

**Comment by lydia-duncan** [comment]
Yes to both

**Pull request merged**